aea/configurations/schemas/definitions.json

```diff
@@ -53,6 +52,17 @@
     "public_id": {
       "type": "string",
       "pattern": "^[a-zA-Z0-9_]*/[a-zA-Z_][a-zA-Z0-9_]*:(0|[1-9]\\d*)\\.(0|[1-9]\\d*)\\.(0|[1-9]\\d*)(?:-((?:0|[1-9]\\d*|\\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\\.(?:0|[1-9]\\d*|\\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\\+([0-9a-zA-Z-]+(?:\\.[0-9a-zA-Z-]+)*))?$"
+    },
+    "version_specifier": {
+      "type": "string",
+      "pattern": "^(( *(~=|==|>=|<=|!=|<|>) *)([1-9][0-9]*!)?(0|[1-9][0-9]*)(\\.(0|[1-9][0-9]*))*((a|b|rc)(0|[1-9][0-9]*))?(\\.post(0|[1-9][0-9]*))?(\\.dev(0|[1-9][0-9]*))?)?$"
+    },
+    "aea_version": {
+      "type": "string",
+      "oneOf": [
+        {"$ref":  "#/definitions/version_specifier"},
+        {"$ref":  "#/definitions/version"}
+      ]
```

Comment:
an `aea_version` is a string that matches one of the following regular expressions:
- A version specifier, e.g. `>=0.2.0`: https://www.python.org/dev/peps/pep-0440/
- semantic versioning format: https://semver.org/#is-there-a-suggested-regular-expression-regex-to-check-a-semver-string

Reply:
Does it make much sense to maintain this backward compatibility for semantic versioning format?

Reply:
@DavidMinarsch the idea is that we support both this:
```aea_version: "0.2.0"```
and this:
```aea_version: ">=0.2.0"

the first one is an alias for `aea_version: "==0.2.0" (or `">=0.2.0"`, depending on how we want to interpret it.)

Reply:
Ok, so then I think it should be interpreted as "==0.2.0"